regress/sql/age_load.sql

```diff
@@ -78,4 +76,27 @@ SELECT * FROM cypher('agload_test_graph', $$
     RETURN u.name, u.region
 $$) AS (result_1 agtype, result_2 agtype);
 
+
+-- TEST DOUBLE LOAD files
+-- These files have been loaded at the beggining with these same labels
+
+SELECT load_labels_from_file('agload_test_graph', 'Country',
```

Comment:
The existing tests creates labels before loading. I was thinking instead of modifying them, let's append the new tests at the end. So, you may load City3, Country3 and has_city3 without creating labels first. Then, for exampe, call the COUNT(*) on both Country and Country3 and they should match.

Reply:
Added:
- Tests without creating labels first, with id field, which are City3, Country3 and has_city3; and 
- tests without creating labels first, without id field, which are City4 and Country4 (there ain't no edge loading without id field).

They both have COUNT(*) on them, which result in 53, 72485 and 72485.